vizro-core/src/vizro/charts/tables/dash_table.py

```diff
@@ -0,0 +1,14 @@
+import pandas as pd
+from dash import dash_table
+
+
+from vizro.models.types import capture
+
+@capture("table")
+def dash_data_table(data_frame: pd.DataFrame, **kwargs):
```

Comment:
Still to discuss some time: what should we call this? `table` or `data_table` or `dash_data_table` are the options I think.

Reply:
Yes, do not have a very strong opinion, but of the top of my hat:
- `table` (very general, and may collide with any future other table additions; really makes the repetition in words stand out, almost feels like precluding any other objects to be inserted)
- `data_table` (similar reasoning as above + somewhat refers to dash (`dash_table.DataTable`), but only very lightly)
- `dash_data_table` or `dash_table` (explicitly refers to dash, which may be useful for guidance, but could feel unnecessarily long)

My gut feeling is probably `dash_table` actually

Reply:
The two relevant table implementations at the moment would be `dash.dash_table.DataTable` and
`dash_ag_grid.AgGrid` so my vote on function names would be `data_table` and `ag_grid`.

Reply:
I'll ask him

Reply:
Joe's vote goes for `dash_data_table` and if we have it in the future `dash_ag_grid`